scripts/machine_translation/average_checkpoint.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import mxnet as mx
+
+mx.npx.set_np()
+
+def get_parser():
+    parser = argparse.ArgumentParser(description='Script to average the checkpoints')
+    parser.add_argument('--prefix', type=str, required=True,
+                        help='Prefix of the checkpoint files, e.g. --prefix ckpt_folder/checkpoint')
+    parser.add_argument('--suffix', type=str, default='',
+                        help='Prefix of the checkpoint files, e.g. --suffix .params')
+    parser.add_argument('--range', type=str, required=True,
+                        help='a piece of python code to represent the checkpoint range, '
+                             'e.g. [1,3,7] range(10) range(4000,5000,2) [5,6]+[7,8]')
```

Comment:
Is it possible to use `nargs='+'`?

Reply:
So we may use regex in the CLI. For example, `python average_checkpoint.py --checkpoints *.params --begin XX --end XX`.